terraform.sample.yml

```diff
@@ -19,6 +19,7 @@
 # `roles/` directory, or by checking the online documentation at
 # microservices-infrastructure.readthedocs.org.
 - hosts: all
+  sudo: yes
```

Comment:
I know it's arguable. My point was that we have sudo for all tasks in a playbook. So, it's to reduce the amount of code. But I think you are right, so I will remove it.